examples/dt/uni05epsilon/control-plane.md

```diff
@@ -0,0 +1,55 @@
+# Configuring networking and deploy the OpenStack control plane
+
+## Assumptions
+
+- A storage class called `local-storage` should already exist.
+
+## Initialize
+
+Switch to the "openstack" namespace
+
+```bash
+oc project openstack
+```
+
+Change to the uni05epsilon directory
+
+```bash
+cd architecture/examples/dt/uni05epsilon
+```
+
+Edit [service-values.yaml](service-values.yaml) and
+[nncp/values.yaml](nncp/values.yaml).
```

Comment:
Given L18 has the user go to `architecture/examples/dt/uni05epsilon`...

```suggestion
Edit [control-plane/service-values.yaml](control-plane/service-values.yaml) and
[control-plane/nncp/values.yaml](control-plane/nncp/values.yaml).
```

Reply:
Hello, thanks for suggestion, but this is still a draft, not ready for review at the moment; we all copy-pasted what we were told to use as a baseline – please ask the question there: https://github.com/openstack-k8s-operators/architecture/pull/83/files

Reply:
Updated.